### PR TITLE
Update IMB_ones_accu.c

### PR DESCRIPTION
**Pull request by VinnitskiV**
Restored barriers.

### CODE REVIEW

File: src_c/IMB_ones_accu.c [hunk 1]
```diff
@@ -150,6 +150,10 @@ Output variables:
     if (c_info->rank < 0)
         *time = 0.;
     else {
+
```
Comment (juhlee-microsoft) [inline src_c/IMB_ones_accu.c:153]: Are we losing the synchronization step in aggregate?

Comment (VinnitskiV) [reply src_c/IMB_ones_accu.c:153]: Hi @juhlee-microsoft , sure - those barriers works not only for nonaggregate mode, all processes do barriers before starting aggregate\nonaggregate benchmark.
We have two independent "if" on line 157 and 185.

Comment (juhlee-microsoft) [reply src_c/IMB_ones_accu.c:153]: Acked. Thanks for letting me know.

**Comment by zhngaj** [comment]
I see IMB has new releases since 2019.6. 
Wondering why this fix did not get merged.

**Review by zhngaj — APPROVED** [review]

**Comment by shijin-aws** [comment]
Hi, we are following up https://github.com/intel/mpi-benchmarks/issues/29, can this fix be merged ?

File: src_c/IMB_ones_accu.c (hunk 1 above)
Comment (EmmanuelBRELLE) [reply src_c/IMB_ones_accu.c:153]: Hi @VinnitskiV  I do not understand why these Barriers only apply to non_aggregation mode. For example for MPI_Get tests both modes have these Barriers: 
https://github.com/intel/mpi-benchmarks/blob/master/src_c/IMB_ones_unidir.c#L297
https://github.com/intel/mpi-benchmarks/blob/master/src_c/IMB_ones_unidir.c#L384

**Pull request closed without merging**